koku/providers/gcp/provider.py

```diff
@@ -11,6 +13,8 @@
 
 LOG = logging.getLogger(__name__)
 
+REQUIRED_IAM_PERMISSIONS = ["bigquery.jobs.create", "bigquery.tables.getData"]
```

Comment:
We can add the `bigquery.tables.list` check when we handle the new account setup flow where the table_id isn't immediately available at source create time.

Reply:
I made sure to include this in the `Check to see if the Table Id is created` issue in the design doc since it will be the issue that has to add that permission.